README.md

```diff
@@ -1,23 +1,33 @@
 
-Data from HiTS (2015) for fast testing of alert production in the LSST DM stack.
+Data from HiTS (2015) for fast testing of alert production in the LSST Science Pipelines.
 
-Contains DECam data from the HiTS (2015) fields `Blind15A_40` and `Blind15A_42`.
+This and other `ap_verify` "datasets" are based on [ap_verify_dataset_template](https://github.com/lsst-dm/ap_verify_dataset_template).
 
+Contains DECam data from the HiTS (2015) fields `Blind15A_40` and `Blind15A_42`:
+
+* Visit 411371, Detectors (ccds) 56 and 60
+* Visit 411420, Detectors (ccds) 5 and 10
+* Visit 419802, Detectors (ccds) 5 and 10
+
+---
+**Note**
 At present, this repository can only be used for Gen 2 processing.
 Support for the new Gen 3 pipeline framework is expected in the future.
 
+---
 Relevant Files and Directories
 -----
 path                  | description
 :---------------------|:-----------------------------
-`raw`                 | Raw, compressed DECam fits images from the HiTS (2015) fields `Blind15A_40` and `Blind15A_42`. 
-`calib`               | DECam master calibs and `wtmap`s from the 2015 HiTS campaign. No raw images. See below for filename information.
-`config`              | Dataset-specific configs to help Stack code work with this dataset.
-`templates`           | Butler repo containing coadded images intended to be used as templates.
-`repo`                | A template for a Butler raw data repository. This directory must never be written to; instead, it should be copied to a separate location, and data ingested into the copy (this is handled automatically by `ap_verify`, see below). Currently contains the appropriate DECam `_mapper` file.
-`refcats`             | Tarballs of Gaia and PS1 reference catalogs in HTM format for regions overlapping both HiTS fields.
+`raw`                 | Raw, compressed DECam fits images from the HiTS (2015) fields `Blind15A_40` and `Blind15A_42`.
+`calib`               | DECam Community Pipeline MasterCalibs and `wtmap`s from the 2015 HiTS campaign. No raw images. See below for filename information.
+`config`              | Dataset-specific configs to help the Science Pipelines work with this dataset.
+`templates`           | Gen2 Butler repo containing good seeing g-band coadds and a skymap. For use as difference imaging templates.
+`repo`                | A template for a Gen2 Butler raw data repository. This directory must never be written to; instead, it should be copied to a separate location, and data ingested into the copy (this is handled automatically by `ap_verify`, see below). Currently contains the appropriate DECam `_mapper` file.
+`refcats`             | Tarballs of Gaia and PS1 reference catalogs in HTM format for regions overlapping all visits in the dataset.
 `dataIds.list`        | List of dataIds for use in running Tasks. Currently set to run all Ids.
+`preloaded`           | Preliminary Gen3 Butler repo containing a skymap as well as the refcats, calibs, and templates described above.
```

Comment:
I'm not sure I'd describe it as "preliminary", which to me sounds like there'll be a better version later. "Seed" or "starting", maybe?